### DISCUSSION TITLE
cover怎么引入本地图片

### DISCUSSION THREAD

**Discussion by itYang1** [hexojs/hexo#5537] [Question / Answer] [answered]
在post文件夹中创建和文件通明的文件夹，想在文件中的cover引入同文件名的文件夹中的图片，为什么失败呢？怎么解决？？
![1111](https://github.com/user-attachments/assets/e6ce13cb-fb48-431a-88f2-851a64783604)
![Uploading 2222.png…]()

**Comment by uiolee** [comment, marked as answer]
https://hexo.io/docs/front-matter

It seems that `cover` is not a item of hexo. It may be related to the theme you use, you should consult the author of your theme.
